example/example/app/urls.py

```diff
@@ -1,12 +1,12 @@
-from django.conf.urls import patterns, include, url
+from django.urls import patterns, include, re_path
 from . import views
 
 urlpatterns = patterns(
     "",
-    url(r"^$", views.home, name="home"),
-    url(r"^page1.html$", views.page1, name="page1"),
-    url(r"^page2.html$", views.page2, name="page2"),
-    url(r"^page3.html$", views.page3, name="page3"),
-    url(r"^page4.html$", views.page4, name="page4"),
-    url(r"^page5.html$", views.page5, name="page5"),
+    re_path(r"^$", views.home, name="home"),
+    re_path(r"^page1.html$", views.page1, name="page1"),
+    re_path(r"^page2.html$", views.page2, name="page2"),
+    re_path(r"^page3.html$", views.page3, name="page3"),
+    re_path(r"^page4.html$", views.page4, name="page4"),
+    re_path(r"^page5.html$", views.page5, name="page5"),
```

Comment:
Would it be better to use the newer format of just...:
```
path("page1.html", views.page1, name="page1"),
```
instead?
Because, there is no need for an actual regex, right?